apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/index.tsx

```diff
@@ -38,12 +42,17 @@ export function EvaluationResults({
     EvaluationResultWithMetadata | undefined
   >(undefined)
   const { data: providerLog } = useProviderLog(selectedResult?.providerLogId)
+  const searchParams = useSearchParams()
+  const page = searchParams.get('page')
+  const pageSize = searchParams.get('pageSize')
   const { data: evaluationResults, mutate } = useEvaluationResultsWithMetadata(
     {
       evaluationId: evaluation.id,
       documentUuid: document.documentUuid,
       commitUuid: commit.uuid,
       projectId: project.id,
+      page,
+      pageSize,
```

Comment:
My guess was that because a different page means a different URL all the client is wipe out. But that's not the case and when you move to page 2 you still see cached page 1 in the client. So I did that the store also is aware of pagination in the cache key so whenever change it make a new fetch

Reply:
This will need to be taken into account when we introduce filters in evaluation results table